### ISSUE TITLE
program not working correctly: "Can't find texture data ..." message

### ISSUE THREAD

**Issue by mizaska** [supadupaplex/PVR2MDL#1]
dragging and dropping the file doesnt seem to work, when doing so this error pops up
![image](https://user-images.githubusercontent.com/81104801/111915139-169b1100-8a75-11eb-95db-34a77bc47d06.png)

**Comment by supadupaplex** [comment]
Hi. Can you upload the model you were trying to convert?

Are you sure that this file is not a sequence group (like barney01.mdl) or model with external textures (is barneyT.mdl file present)?

**Comment by mizaska** [comment]
> Hi. Can you upload the model you were trying to convert?
> 
> Are you sure that this file is not a sequence group (like barney01.mdl) or model with external textures (is barneyT.mdl file present)?

yes, barneyT is present and its not a sequence group, keep in mind however that this is a different prototype im using (build april 20 2001) then the one thats more common online (build july 13 2001) 
[barneydc.zip](https://github.com/supadupaplex/PVR2MDL/files/6191839/barneydc.zip)

**Comment by supadupaplex** [comment]
If there is a barneyT.mdl present that means that barney.mdl file has no textures and the program just says that there is nothing to be done for this particular file because the only thing that is different between PC and DC models is texture format. What you need to do is to convert the barneyT.mdl file instead because all textures are stored in this file.

This program does not try to look for other files that may be related to the input file (like hlmv does).

There is also no need to drag and drop individual files on .exe, you can just put "pvr2mdl.exe" and "convert_all_to_mdl.bat" files into the models diectory and then start the .bat script: when it's done all models in the directory will be in PC compatible format.
